api/internal/utils/consts/error.go

```diff
@@ -16,13 +16,33 @@
  */
 package consts
 
-import "github.com/shiningrush/droplet/data"
+import (
+	"errors"
+
+	"github.com/shiningrush/droplet/data"
+)
 
 const (
 	ErrBadRequest = 20001
 	ErrForbidden  = 20002
 )
 
+const (
+	UpstreamIDNotFound = "upstream id: %s not found"
```

Comment:
Could `upstream` be changed to a variable?

Reply:
Yes. We can.

Reply:
Updated.